version.properties

```diff
@@ -0,0 +1,2 @@
+#Wed Dec 16 14:41:05 PST 2020
+VERSION=0.5.0-SNAPSHOT
```

Comment:
I think the current version is different to this one.
(0.5.1?)

In any case, they should be always ascendent :-)

Reply:
Yep I am going to try things out with this first, then I will definitely bump it.